plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -383,6 +383,10 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
             throw new IllegalArgumentException(format("Unexpected column for table '%s$query': %s", table.getIndex(), column.getName()));
         }
 
+        if (BuiltinColumns.NAME_TO_COLUMN.containsKey(column.getName())) {
+            return BuiltinColumns.NAME_TO_COLUMN.get(column.getName()).getMetadata();
+        }
+
         return ColumnMetadata.builder()
```

Comment:
What is the benefit of above if?

Reply:
Before this change `getColumnMetadata` created wrong metadata for `BuiltinColumns`. `BuiltinColumns` have `.setHidden(true)` for their metadata.

Reply:
So as I understand that leaks and should be visible to user as some kind of bug. Can you see what it caused? Maybe builtin column was exposed via `SHOW CREATE TABLE`? Can you have a test for this?

Reply:
found during #6745
> Can you have a test for this?

I'm not sure.